app/scripts/dimApp.i18n.js

```diff
@@ -112,34 +214,132 @@
+          FilterSets: "Sets filtern",
+          AdvancedOptions: "Erweiterte Optionen",
+          Fast: "Schnelle",
+          Full: "Vollständige",
+          ProcessingMode: "Berechnung",
+          ProcessingModeHelpFast: "Nur die beste Ausrüstung wird einbezogen.",
+          ProcessingModeHelpFull: "Bezieht die ganze Ausrüstung mit ein.",
+          Scaled: "Skaliertes",
+          Current: "Aktuelles",
+          LightMode: "Lichtlevel",
+          LightModeHelpScaled: "Berechnet Loadouts, bei denen alle Gegenstände Lichtlevel 350 besitzen.",
+          LightModeHelpCurrent: "Berechnet Loadouts mit dem aktuellen Lichtlevel der Gegenstände.",
+          LBIncludeRare: "Seltene (blaue) Gegenstände einbeziehen",
+          LBhelp: "Brauchst du Hilfe?",
+          LBEquip: "Am aktuellen Charakter anlegen.",
+          LBShowAllConfigs: "Zeige alle Varianten",
+          LBHideAllConfigs: "Verstecke alle Varianten",
+          LBLoading: "Lade die besten Sets ({{percentage}}%)",
+          LBVendor: "Gegenstände von Händlern einschließen",
+          LBExclude: "Ignorierte Gegenstände",
+          LBExcludeHelp: "Benutze Shift + Klick bei einem Gegenstand (oder ziehe ihn in dieses Feld) um Sets ohne diesen Gegenstand zu generieren.",
+          LBLockPerk: "Perk festlegen",
+          LBMissing1: "Es fehlen legendäre oder exotische Gegenstände, um ein vollständiges Set zu generieren!",
+          LBMissing2: "Es fehlen seltene, legendäre oder exotische Gegenstände, um ein vollständiges Set zu generieren",
+          ManifestBuild: "Lege Destiny Datenbank an",
+          ManifestError1: "Fehler beim Laden von Informationen: ",
+          ManifestError2: "App neu laden, um es nochmals zu versuchen.",
+          ManifestDownload: "Lade neueste Daten von Bungie herunter",
+          ManifestLoad: "Lade gespeicherte Daten",
+          ManifestUnzip: "Entpacke neueste Daten",
+          ManifestSave: "Speichere neueste Daten",
+          VendorsLoad: "Lade Händler",
+          ArmorAndWeapons: "Panzerung & Waffen",
+          ShipsAndVehicles: "Schiffe & Fahrzeuge",
+          Consumables: "Verbrauchsgegenstände",
+          Bounties: "Beutezüge",
+          ShadersAndEmblems: "Shader & Embleme",
+          Emotes: "Emotes",
+          RefreshDestinyData: "Aktualisiere Destiny Daten",
+          Filters: "Filter",
+          Language: "Sprache (lade DIM neu zum Übernehmnen)",
+          HideUnfiltered: "Zeige nur die Suchergebnisse beim Filtern",
+          HideUnfilteredHelp: "Gegenstände, die nicht zum Filter passen, werden ausgeblendet.",
+          AlwaysShowDetails: "Zeige immer Details der Gegenstände",
+          AlwaysShowDetailsHelp: "Ein Klick auf einen Gegenstand öffnet ein Popup, welches erweitert werden kann, um Details zu Statistiken und Perks zu zeigen. Diese Option wird immer diese Details zeigen, wenn du auf einen Gegenstand klickst.",
+          EnableAdvancedStats: "Aktivieren der erweiterten Statusvergleichsfunktionen",
```

Comment:
@Korben85 @crulplay can we shorten this sting by ~4 characters? Is it still understandable is we remove `der`? Or any other ideas?